modules/core/shared/src/main/scala/gem/Observation.scala

```diff
@@ -14,17 +18,45 @@ import cats.implicits._
  * @group Program Model
  */
 final case class Observation[+S, +D](
-  id: Observation.Id,
   title: String,
   staticConfig: S,
   steps: List[D])
 
 object Observation {
 
+  /** A positive, non-zero integer for use in ids. */
+  sealed abstract case class Index(toInt: Int) {
+    def format: String =
+      s"$toInt"
```

Comment:
Just curious if there's some reason you did it this was instead of `toInt.toString`?

Reply:
I bet because `WartRemover` complains about `toString`

Reply:
i've been defining lots of `Show.fromToString` to get around that `WartRemover` feature. I don't think it's that helpful